db/seeds/user_groups.seeds.rb

```diff
@@ -233,14 +233,20 @@
     is_hidden: true,
     metadata: GroupsMetadataTeamsCommittees.find_by!(friendly_id: 'wst_admin'),
   )
-
   UserGroup.create!(
     name: 'WCT China',
     group_type: :teams_committees,
     is_active: true,
     is_hidden: true,
     metadata: GroupsMetadataTeamsCommittees.find_by!(friendly_id: 'wct_china'),
   )
+  UserGroup.create!(
+    name: 'WCA Data Protection Committee',
+    group_type: :teams_committees,
+    is_active: false,
+    is_hidden: false,
```

Comment:
uh oh

Reply:
It was intentional actually. WDPC 'team' is hidden because it is no longer active but there was no way in teams table to set active status. Here, `is_active` solves that. Hence, if we decide to show past roles of a user in a public page, we have to show their WDPC role (if exists) as well and for that the group should not be hidden.